po/oc.po

```diff
@@ -0,0 +1,705 @@
+msgctxt "Name of the color set from the xkcd color survey"
+msgid "xkcd"
+msgstr "xkcd"
+
+#: src/widgets/preferences_window.rs:189
+msgid "Show color names from the xkcd color survey"
+msgstr ""
+
+#: src/widgets/preferences_window.rs:328
+msgid "Move Up"
+msgstr "Montar"
+
+#: src/widgets/preferences_window.rs:330
+msgid "Move Down"
+msgstr "Davalar"
+
+#: src/widgets/preferences_window.rs:453
+msgid "Hex-Code"
+msgstr "Còdi HEX"
+
+#: src/widgets/preferences_window.rs:458
+msgid "RGB"
+msgstr "RVB"
+
+#: src/widgets/preferences_window.rs:461
+msgid "HSL"
+msgstr "TSL"
+
+#: src/widgets/preferences_window.rs:464
+msgid "HSV"
+msgstr "TSV"
+
+#: src/widgets/preferences_window.rs:468
+msgid "CMYK"
+msgstr "CMJN"
+
+#: src/widgets/preferences_window.rs:473
+msgid "XYZ"
+msgstr "XYZ"
+
+#: src/widgets/preferences_window.rs:477
+msgid "CIELAB"
+msgstr ""
+
+#: src/widgets/preferences_window.rs:482
+msgid "HWB"
+msgstr "TBN"
+
+#: src/widgets/preferences_window.rs:486
+msgid "CIELCh / HCL"
+msgstr ""
+
+#: src/widgets/preferences_window.rs:492
+msgid "Name"
+msgstr "Nom"
+
+#: src/widgets/preferences_window.rs:495 src/window.rs:438 src/window.rs:461
+#: src/window.rs:624 src/window.rs:695
+msgctxt "Information that no name for the color could be found"
+msgid "Not named"
+msgstr "Sens nom"
+
+#: src/widgets/preferences_window.rs:500
+msgid "LMS"
+msgstr "LMS"
+
+#: src/widgets/preferences_window.rs:504
+msgid "Hunter Lab"
+msgstr ""
+
+#. Translators: This should not be translate, Please enter your credits here instead (format: "Name https://example.com" or "Name <email@example.com>", no quotes)
+#: src/widgets/about_window.rs:51
+msgid "translator-credits"
+msgstr "Quentin PAGÈS"
+
+#: src/widgets/about_window.rs:111
+msgctxt ""
+"The changelog of the current version. Should be similar to the CHANGELOG.md "
+"file at the project root. Please ensure that the tags have a matching "
+"closing tag. Since xgettext does not recognize rust multiline strings, this "
+"should be on a single line using line breaks (\\n) for new lines."
+msgid ""
+"<p>A minor release with many small QoL improvements.</p><p>Added:</p>\n"
+"<ul><li>Show a toast with an Undo option when clearing the history</"
+"li><li>Use Ctrl-Q to close the window</li><li>Use a default minimum window "
+"size, content should no longer unnecessarily cut off</li><li>New icon by "
+"@bertob</li><li>Export palettes as Adobe Swatch Exchange (ASE), hex, PAL, "
+"Paint.NET (txt) files</li><li>Minor changes to the store metadata</li></"
+"ul><p>Changed:</p>\n"
+"<ul><li>Hunter-Lab now displays 0 instead of NaN</li><li>Improved tooltips "
+"for copying colors</li><li>Palette window now uses a button with a more "
+"explicit text instead of an icon</li><li>Reworded project description</"
+"li><li>Updated all screenshots</li></ul><p>Fixed:</p>\n"
+"<ul><li>Separator is now hidden when the history list is hidden</"
+"li><li>Improved editing experience when change the Hex color/name</li></ul>"
+msgstr ""
```

Comment:
This one should technically be translated, but as it will be rewritten for the next release, it doesn't matter if it's not.